x-pack/plugins/observability_onboarding/server/routes/custom_logs/get_logs_count.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ElasticsearchClient } from '@kbn/core/server';
+
+export async function getLogsCount({
```

Comment:
This will only be valid if the combination of `dataset` and `namespace` the user picked don't have logs already, as soon as the dataset name is the same we will be giving false positives here.
Could we assume that case as corner case?

Reply:
Follow up for this topic: https://github.com/elastic/kibana/issues/159387